visidata/basesheet.py

```diff
@@ -194,7 +194,7 @@ def execCommand(self, longname, vdglobals=None, keystrokes=None):
 
         cmd = self.getCommand(longname or keystrokes)
         if not cmd:
-            vd.warning('no command for %s' % (longname or keystrokes))
+            vd.fail('no command for %s' % (longname or keystrokes))
```

Comment:
I have a vague recollection that `fail` was not desired in replay at one time.  Maybe it was that some commands would `fail`, but you wanted them to continue anyway?  Like `select-regex`, if nothing matches, you can still `delete-selected` as a no-op, but you don't want to abort the replay entirely.

That's almost entirely tangential to this, but it makes me wonder if something like this should be `error` instead of `fail`?  There's almost no difference between them--error is red while fail is yellow.  But we could use them more semantically, in case we want to catch `fail`s and continue the replay, vs let `error`s abort the replay.

Either way, I agree that an unknown command should abort the replay.